Modules/Core/Common/test/itkAnnulusOperatorTest.cxx

```diff
@@ -31,12 +32,18 @@ itkAnnulusOperatorTest(int, char *[])
   using PixelType = float;
   using OperatorType = itk::AnnulusOperator<PixelType, Dimension>;
 
-
   OperatorType normalizedAnnulus;
+
+  ITK_EXERCISE_BASIC_OBJECT_METHODS((&normalizedAnnulus), AnnulusOperator, NeighborhoodOperator);
+
+
   normalizedAnnulus.NormalizeOn();
   normalizedAnnulus.SetInnerRadius(3);
   normalizedAnnulus.SetThickness(2);
-  normalizedAnnulus.BrightCenterOff();
+
+  auto brightCenter = false;
```

Comment:
Changed in 8449d1a. Not sure how consistent we've been concerning this, and I'd say that the use of `auto` has gained popularity in the code base lately. Probably being clear about this in the ITK SW Guide would help (the testing section was written a while ago).

Even in the CI builds pass, not sure if the dashboard sites will complain since most of the setter methods do not have the `const` keyword.

I've got a separate branch to be submitted once this one gets merged transitioning to the Set/Get macro methods.